src/cloud_common/schemas/service_config/aos_config.py

```diff
@@ -0,0 +1,167 @@
+        datetime,
+        Field(
+            alias='created',
+            description='Timestamp when Aos service was created.',
+        ),
+    ]
+
+    author: Annotated[
+        Optional[str],
+        Field(
+            alias='author',
+            default=None,
+            description='Aos service author.',
+        ),
+    ]
+
+    balancing_type: Annotated[
+        Literal[
+            'enabled',
+            'disabled',
+        ],
+        Field(
+            alias='balancingType',
+            default='enabled',
+            description='Balancing type. `disabled` means total prohibition from balancing to other nodes.`',
+        ),
+    ]
+
+    service_provider: Annotated[
+        str,
+        Field(
+            alias='serviceProvider',
+            description='Aos service provider unique identifier.',
+        ),
+    ]
+
+    hostname: Annotated[
+        Optional[str],
+        Field(
+            alias='hostname',
+            default=None,
+            description='The hostname of the Aos service. The FQDN is {hostname].{service_provider}.',
+        ),
+    ]
+
+    runners: Annotated[
+        Optional[list[Literal['runc', 'crun', 'xrun']]],
+        Field(
+            alias='runners',
+            default=None,
+            description='Aos service allowed runner types. Absense means ["runc", "crun"].',
+        ),
+    ]
+
+    run_parameters: Annotated[
+        Optional[RunParameters],
+        Field(
+            alias='runParameters',
+            default=None,
+            description='Run parameters for the Aos service.',
+        ),
+    ]
+
+    offline_ttl: Annotated[
+        Optional[timedelta],
+        Field(
+            alias='offlineTTL',
+            default=None,
+            description="""\
+TTL (allowed time) to run service when unit in offline mode.
+If value is absent service will live on an unit forever.
+Format: ISO8601 duration.""",
+            examples=['PT1M', 'PT7D']
+        ),
+    ]
+
+    devices: Annotated[
+        Optional[UnitDevice],
+        Field(
+            alias='devices',
+            default=None,
+            description='List of needed or requested devices.'
+        ),
+    ]
+
+    resources: Annotated[
+        Optional[list[str]],
+        Field(
+            alias='resources',
+            default=None,
+            description='List of needed resources.',
+            examples=['bluetooth', 'system-dbus'],
+        )
+    ]
+
+    allowed_connections: Annotated[
+        Optional[list[dict]],
+        Field(
+            alias='allowedConnections',
+            default=None,
+            description="""\
+List of allowed network connections.
+Format of connection string: {service_uid}/[port|port_range]/[tcp|udp]""",
+            examples=[
+                '9931560c-be75-4f60-9abf-08297d905332/8087:8088/tcp',
+                '9931560c-be75-4f60-9abf-08297d905332/1515/udp',
+            ]
+        ),
+    ]
+
+    exposed_ports: Annotated[
```

Comment:
This also new, and looks the similar to allowed_connections in case service_uid is omitted.